integration-tests/tests/src/tests/sync/flexible.ts

```diff
@@ -1501,6 +1506,8 @@ describe.skipIf(environment.missingServer, "Flexible sync", function () {
         mutableSubs.add(newRealm.objects(FlexiblePersonSchema.name).filtered("age > 30"));
       });
 
+      await newRealm?.syncSession?.uploadAllLocalChanges();
```

Comment:
Again I don't think this one should be necessary as we didn't create any local objects prior to calling it

Reply:
Removed